public/characterCreator.html

```diff
@@ -1 +1,96 @@
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <style>
+
+    </style>
+
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="">
+</head>
+
+<body>
+    <!--[if lt IE 7]>
+            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+    <form id="characterCreator" name="characterCreator" action="../creator.js" method="POST">
+        <label>Name</label>
+        <input type="text" id="name" name="name"><br><br>
+        <label>Nickname</label>
+        <input type="text" id="nickname" name="nickname"><br><br>
+        <label>Level</label>
+        <input type="number" id="lvl" name="lvl" value="1"><br><br>
+        <label>Stat Points</label>
+        <input type="number" id="statpoints" name="statpoints" value="0"><br><br>
+        <label>Experience</label>
+        <input type="number" id="xp" name="xp" value="0" ><br><br>
+        <label>IQ</label>
+        <input type="number" id="iq" name="iq" value="0" class="stats"><br><br>
+        <label>Mental Endurance</label>
+        <input type="number" id="mEndurance" name="mEndurance" value="0" class="stats"><br><br>
+        <label>Mental Affinity</label>
+        <input type="number" id="mAffinity" name="mAffinity" value="0" class="stats"><br><br>
+        <label>Physical Strength</label>
+        <input type="number" id="pStrength" name="pStrength" value="0" class="stats"><br><br>
+        <label>Physical Prowess</label>
+        <input type="number" id="pProwess" name="pProwess" value="0" class="stats"><br><br>
+        <label>Physical Endurance</label>
+        <input type="number" id="pEndurance" name="pEndurance" value="0" class="stats"><br><br>
+        <label>Speed</label>
+        <input type="number" id="speed" name="speed" value="0" class="stats"><br><br>
+        <label>Wealth</label>
+        <input type="number" id="wealth" name="wealth" value="0"><br><br>
+        <label>Power Name</label>
+        <input type="text" id="powername" name="powername"><br><br>
+        <label>Power Description</label>
+        <input type="text" id="powerdescription" name="powerdescription"><br><br>
+        <input type="submit" value="Create">
+    </form>
+    <button onclick="preview()">Preview</button>
+    <script type="text/javascript">
+    function preview(){
+      const elements = document.getElementsByClassName("stats");
+      console.log(elements);
+      let y = calculateDeficit(elements);
+
+      document.getElementById('deficitpoints').innerHTML = y;
+
+    }
+    function generateStatCost(x){
```

Comment:
you can do this without let

Reply:
i.e. in a way where you don't have to declare variables

Reply:
Maybe did it correctly?

Reply:
Huh?